checks/raw/shell_download_validate.go

```diff
@@ -696,6 +697,93 @@ func isChocoUnpinnedDownload(cmd []string) bool {
 	return true
 }
 
+func isUnpinnedNugetCliInstall(cmd []string) bool {
+	// looking for command of type nuget install ...
+	if len(cmd) < 2 {
+		return false
+	}
+
+	// Search for nuget commands.
+	if !isBinaryName("nuget", cmd[0]) {
```

Comment:
do we need to check for `nuget.exe` as well?

Reply:
it's not uncommon that people rename nuget.exe occassionally as well, usually by appending the version

Reply:
added thanks!

Reply:
Thanks everyone. Resolving comment.